pkg/infra/teams/team3/agent3.go

```diff
@@ -71,8 +70,10 @@ func (a *AgentThree) UpdateInternalState(baseAgent agent.BaseAgent, _ *commons.I
 	if enable {
 		a.UpdatePersonality(baseAgent)
 	}
-	//fmt.Println(a.CalcBordaScore(baseAgent))
-	fmt.Println(a.SocialCapital(baseAgent))
+	// fmt.Println(a.CalcBordaScore(baseAgent))
+	a.CalcBordaScore(baseAgent)
+	// fmt.Println(a.SocialCapital(baseAgent))
+	a.SocialCapital(baseAgent)
```

Comment:
Why aren't the return values used?

Reply:
left them up to @Massoud26's implementation, this PR is just fixing bugs regarding the maps that weren't working

Reply:
sounds fair enough :)